modules/tinymce/src/core/test/ts/browser/ModeTest.ts

```diff
@@ -108,4 +109,14 @@ describe('browser.tinymce.core.ModeTest', () => {
     assertMode(editor, 'readonly');
     assertBodyClass(editor, 'mce-content-readonly', true);
   });
+
+  it('TINY-6800: even in readonly mode copy event should be dispatched', () => {
```

Comment:
This would be better suited in `ReadOnlyModeTest` instead of here.

Reply:
moved